elements/auras.lua

```diff
@@ -194,25 +205,49 @@ local function updateIcon(element, unit, index, offset, filter, isDebuff, visibl
 
 		* show - indicates whether the aura button should be shown (boolean)
 		--]]
-		local show = (element.CustomFilter or customFilter) (element, unit, button, name, texture,
-			count, debuffType, duration, expiration, caster, isStealable, nameplateShowSelf, spellID,
-			canApply, isBossDebuff, casterIsPlayer, nameplateShowAll,timeMod, effect1, effect2, effect3)
-
-		if(show) then
+		local show = (element.CustomFilter or customFilter)(
+			element,
+			unit,
+			button,
+			name,
+			texture,
+			count,
+			debuffType,
+			duration,
+			expiration,
+			caster,
+			isStealable,
+			nameplateShowSelf,
+			spellID,
+			canApply,
+			isBossDebuff,
+			casterIsPlayer,
+			nameplateShowAll,
+			timeMod,
+			effect1,
+			effect2,
+			effect3
+		)
```

Comment:
Great example where splitting arguments into newlines can be a bad idea.

Reply:
There is a plus here - counting arguments is way easier because once can subtract the line numbers :)
But I see what you mean.